docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx

```diff
@@ -1,465 +1,489 @@
+  Other Teleport services provide access to remote desktops, Kubernetes
+  clusters, applications, and databases.
 
-This guide will run a Teleport cluster consisting of the Auth Service, Proxy
-Service, and SSH Service, and interact with the cluster using Teleport's
-client-side tools:
+You will install the `teleport` binary, which runs runs all three of these
+services by default.
+
+You will also use the following client tools:
 
 | Tool | Description |
 | - | - |
-| tctl | Cluster administration tool used to invite Nodes to a cluster and manage user accounts. |
-| tsh | Allows users to authenticate and access resources via their local machine. `tsh`'s SSH functionality is similar in principle to OpenSSH's `ssh`. Users can log in to remote Nodes, list and search for Nodes in a cluster, securely upload/download files, etc. |
-| browser | You can use your web browser to log in to any Teleport Node by opening `https://<proxy-host>:3080`. |
+| `tctl` | Cluster administration tool used to perform tasks such as inviting Nodes to a cluster and managing user accounts. |
+| `tsh` | Allows users to authenticate and access resources via their local machine. |
+| Web UI | You can use the Teleport Web UI to access resources in your cluster by navigating to the public address of your Teleport Proxy Service in your browser. |
 
+(!docs/pages/includes//cloud/call-to-action.mdx!)
 
 ## Prerequisites
 
-You will need to have access to the [customer portal](https://dashboard.gravitational.com/web/login)
-to download the software. You will also need three computers: two servers and
-one client (probably a laptop) to complete this tutorial. Let's assume the servers have
-the following DNS names and IPs:
+- A Teleport Enterprise account. If you do not have one, use our [signup
+  form](https://goteleport.com/signup/enterprise/) to schedule a demo with the
+  Teleport Sales Team.
+- A Linux machine with only port `443` open to ingress traffic. You must be able
+  to install and run software on the machine. Either configure access to your
+  machine via SSH for the initial setup (and open an SSH port in addition port
+  `443`) or enter the commands in this guide into an Amazon EC2
+  [user data script](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html),
+  Google Compute Engine
+  [startup script](https://cloud.google.com/compute/docs/instances/startup-scripts),
+  or similar.
+- A two-factor authenticator app such as [Authy](https://authy.com/download/),
+  [Google Authenticator](https://www.google.com/landing/2step/), or [Microsoft
+  Authenticator](https://www.microsoft.com/en-us/account/authenticator).
+- `python3` installed on your Linux machine. We will use this to run a simple
+  HTTP file server, so you can use another HTTP server if you have one
+  installed.
+
+You must also have one of the following:
+- A registered domain name.
+- An authoritative DNS nameserver managed by your organization, plus an existing
```

Comment:
What does "plus an existing certificate authority" mean?

Reply:
Made this more specific